### DISCUSSION TITLE
org-mode like advanced folding inclusive nice keybindings

### DISCUSSION THREAD

**Discussion by sotte** [nvim-neorg/neorg#906] [Ideas]
org-mode and most org-mode clones have nice folding support. Press `<tab>` to fold/unfold current node (heading, source code, properties, etc).
```
,-> FOLDED -> CHILDREN -> SUBTREE --.
'-----------------------------------'
```

`<s-tab>` to globally toggle the document between
```
,-> OVERVIEW -> CONTENTS -> SHOW ALL --.
'--------------------------------------'
```

Details: https://orgmode.org/manual/Global-and-local-cycling.html

Are there any plans to support something like this? Or are there already some bindings to support this (I did not find anything)?

**Comment by jghauser** [comment]
One way to get this behaviour is with the [fold-cycle.nvim](https://github.com/jghauser/fold-cycle.nvim) plugin (I'm the author).
